packages/app/cypress/e2e/runner/reporter-ct-mount-hover.cy.ts

```diff
@@ -0,0 +1,46 @@
+import type { fixtureDirs } from '@tooling/system-tests'
+
+type ProjectDirs = typeof fixtureDirs
+
+const PROJECTS: {projectName: ProjectDirs[number], test: string}[] = [
+  { projectName: 'angular-15', test: 'app.component' },
+  { projectName: 'react-vite-ts-configured', test: 'App.cy' },
+  { projectName: 'react18', test: 'App.cy' },
+  { projectName: 'create-react-app-configured', test: 'App.cy' },
+  { projectName: 'vueclivue3-configured', test: 'HelloWorld.cy' },
+  { projectName: 'vuecli4-vue2', test: 'HelloWorld.cy' },
+  { projectName: 'vueclivue2-configured', test: 'HelloWorld.cy' },
+  { projectName: 'nuxtjs-vue2-configured', test: 'Tutorial.cy' },
+]
+
+for (const { projectName, test } of PROJECTS) {
+  describe(`CT Mount ${projectName}`, { viewportWidth: 1500, defaultCommandTimeout: 10000 }, () => {
+    beforeEach(() => {
+      cy.scaffoldProject(projectName)
+      cy.findBrowsers()
+    }),
+    it(`While hovering on Mount(), shows component on AUT for ${projectName}`, () => {
+      if (`${projectName}` === 'react18') {
+        cy.openProject(projectName, ['--config-file', 'cypress-vite.config.ts'])
+        cy.startAppServer('component')
+        cy.visitApp()
+        cy.contains(`${test}`).click()
+        cy.waitForSpecToFinish()
+        cy.get('.collapsible-header-inner:first').click().get('.command.command-name-mount > .command-wrapper').realHover().then(() => {
+          cy.get('iframe.aut-iframe').its('0.contentDocument.body').then(cy.wrap).within(() => {
```

Comment:
I just added two more assertions. The first one ensures that the content body contains at least 2 children i.e. `[data-cy-root]` and the component itself. The second assertions ensures that we can find`[data-cy-root]` within the content body.

Reply:
Hmm, these tests all still pass for me against the old code. There could be a few things going on.

- the component gets mounted into the `data-cy-root` element, so I think we need to check its children, siblings inserted beside it in `0.contentDocument.body` are likely elements that do indicate the component mounted, but they aren't the component.

- I'm not sure about the realHover, it seems I get different results against the old code when I use realHover vs just `click` to pin the snapshot ... Real hover passes on the old code, even when I update the test to check for children of `data-cy-root`. Pinning the snapshot with `Click` fails as expected on the old code and passes on the new code. 

Is it possible realHover is not having the intended effect and we are capturing the end state of the test, not the snapshot from hovering the command? If it's more reliable, even though the ticket was for hover, I think it's safe to say that if the pinned snapshot has the correct DOM content, the hover is fine, so you might be able to switch to click.

To switch between versions of the mount command, I am running `yarn build-prod` on packages/vue2 in develop, and then switching to this branch and running the new e2e test against that old version of the code, and I've removed all but `{ projectName: 'nuxtjs-vue2-configured', test: 'Tutorial.cy' }` from `PROJECTS`.

Reply:
Here's my variation that fails on the old code and passes on the new: 

```js
 cy.get('.command.command-name-mount > .command-wrapper').click().then(() => {
          cy.get('iframe.aut-iframe').its('0.contentDocument.body').then(cy.wrap).within(() => {
            cy.get('[data-cy-root]').children().should('have.length.at.least', 1)
          })
        })
```

It's probably possible to debug the hover some more - maybe the assertion about children runs before Cy has had a chance to react to the hover and update the DOM & we just need to wait for something.

The main thing is to get these tests to fail against the old code for the commands you fixed.

Reply:
As discussed, I will use `click()` instead of `realHover()` or `trigger('mouseover')` as they are not testing the intended functionality. As you mentioned, the tests seems to be failing for the old code when using `click()` which should be the intended outcome as the fix is not implemented in the old code.
As far as asserting the children of `data-cy-root`, this will not always work because some frameworks such as angular do not mount the component under test as a child of `data-cy-root`. Therefore, testing if the content-body has 2 children should suffice.

Reply:
I have updated the tests. All `vue2` and `react18` tests fail when using the old code while all tests pass with the updated `mount` in the current branch.